admin/howto/create-billing-account.md

```diff
@@ -0,0 +1,73 @@
+# Provision a paid cloud provider account for 2i2c
+
+For 2i2c to build & manage your JupyterHub, it needs *full* access to a
+cloud provider account on [Google Cloud Platform](https://cloudn.google.com),
+[Amazon Web Services][https://aws.amazon.com], or [Azure](https://azure.microsoft.com).
+This page will provide guidance on how to do that.
+
+## Google Cloud Platform
+
+Currently, Google Cloud Platform (GCP) is 2i2c's preferred cloud provider. 2i2c
```

Comment:
Maybe a few words about why this is the preferred cloud provided to 2i2c will be helpful to get more context.

Reply:
Je, I noticed this one was already merged, I still think the comment is relevant (feel free to point me to another place where this is discussed... if that place exists).

Reply:
ah, it's mostly because that's where we have the most experience. I'll start an issue to discuss this